amundsen_application/static/js/ducks/tableMetadata/sagas.ts

```diff
@@ -34,15 +35,22 @@ import {
 } from './types';
 
 export function* getTableDataWorker(action: GetTableDataRequest): SagaIterator {
+  const { key, searchIndex, source } = action.payload;
   try {
-    const { key, searchIndex, source } = action.payload;
     const { data, owners, statusCode, tags } = yield call(
       API.getTableData,
       key,
       searchIndex,
       source
     );
     yield put(getTableDataSuccess(data, owners, statusCode, tags));
+
+    try {
```

Comment:
If we succeed in getting the table metadata, this will now fetch the related dashboards.